### DISCUSSION TITLE
MongoDB Support

### DISCUSSION THREAD

**Discussion by Digital39999** [diced/zipline#273] [Ideas]
Since Prisma supports MongoDB, and its free, it would be actually neat to add support for it, doesn't require much work, and it would be useful, also it could be used as file storage instead of local. Thanks for reading.

**Comment by diced** [comment]
This will not be added into Zipline, as it is a major pain to support multiple database providers within Prisma. Trust me, I have tried this before and it is anything but fun. If you would like, feel free to make a fork of Zipline with mongo support, although I will be unable to help with any issues that arise while doing so- you are on your own.

**Comment by Digital39999** [comment]
Thanks, maybe I'll do it one day.
